yoti-sdk-impl/src/main/java/com/yoti/api/client/qrcode/extension/SimpleExtensionBuilder.java

```diff
@@ -0,0 +1,69 @@
+package com.yoti.api.client.qrcode.extension;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.yoti.api.client.spi.remote.util.Validation;
+
+public class SimpleExtensionBuilder extends ExtensionBuilder {
+
+    private String type;
+    private Map<String, ?> content;
+
+    @Override
+    protected ExtensionBuilder createExtensionBuilder() {
+        return new SimpleExtensionBuilder();
+    }
+
+    public ExtensionBuilder withType(String type) {
+        this.type = type;
+        return this;
+    }
+
+    public ExtensionBuilder withContent(Map<String, ?> content) {
+        this.content = content;
+        return this;
+    }
+
+    public ExtensionBuilder forLocationConstraint(double latitude, double longitude, double radius, double maxUncertainty) {
```

Comment:
The comments above allow you to not have to add more code here as we start supporting more and more extensions.
Instead, you would be creating a new typed builder.